src/main/java/com/hillel/items_exchange/controller/ImageController.java

```diff
@@ -0,0 +1,37 @@
+package com.hillel.items_exchange.controller;
+
+import com.hillel.items_exchange.dto.ImageDto;
+import com.hillel.items_exchange.service.ImageService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/products")
```

Comment:
I don't think this path fits to the controller. You should change it.

Reply:
I removed RequestMapping and changed GetMapping. I am not certain how to do it right.